src/components/ResourceList.tsx

```diff
@@ -16,15 +16,107 @@ import {
+    let icon: React.ReactNode;
+
+    if (acceptedCondition) {
+      labelText = 'Accepted';
+      color = 'green';
+      icon = <CheckCircleIcon />;
+    } else if (programmedCondition) {
+      labelText = 'Programmed';
+      color = 'blue';
+      icon = <CheckCircleIcon />;
+    } else if (conflictedCondition) {
+      labelText = 'Conflicted';
+      color = 'red';
+      icon = <ExclamationTriangleIcon />;
+    } else if (resolvedRefsCondition) {
+      labelText = 'Resolved Refs';
+      color = 'blue';
+      icon = <CheckCircleIcon />;
+    } else {
+      labelText = 'Unknown';
+      color = 'orange';
+      icon = <ExclamationTriangleIcon />;
+    }
+
+    return <Label icon={icon} color={color}>{labelText}</Label>;
   }
-  return obj.status.conditions
-    .map((condition: any) => `${condition.type}=${condition.status}`)
-    .join(',');
+
+  if (!obj.status || !obj.status.conditions || obj.status.conditions.length === 0) {
+    // No status/conditions
+    return (
+      <Label icon={<OutlinedHourglassIcon />} color="cyan">
+        Creating
+      </Label>
+    );
+  }
+  const enforcedCondition = obj.status.conditions.find((cond: any) => cond.type === 'Enforced' && cond.status === 'True');
+  const acceptedCondition = obj.status.conditions.find((cond: any) => cond.type === 'Accepted' && cond.status === 'True');
+  const acceptedConditionFalse = obj.status.conditions.find((cond: any) => cond.type === 'Accepted' && cond.status === 'False');
+  const overriddenCondition = obj.status.conditions.find((cond: any) => cond.type === 'Overridden' && cond.status === 'True');
+  const conflictedCondition = obj.status.conditions.find((cond: any) => cond.reason === 'Conflicted' && cond.status === 'False');
+  const targetNotFoundCondition = obj.status.conditions.find((cond: any) => cond.reason === 'TargetNotFound' && cond.status === 'False');
+  const unknownCondition = obj.status.conditions.find((cond: any) => cond.reason === 'Unknown' && cond.status === 'False');
+
+  let labelText: string;
+  let color: 'blue' | 'green' | 'red' | 'orange' | 'grey' | 'purple' | 'cyan';
+  let icon: React.ReactNode;
+
+  if (enforcedCondition) {
+    labelText = 'Enforced';
+    color = 'green';
+    icon = <CheckCircleIcon />;
+  } else if (overriddenCondition) {
+    labelText = 'Overridden (Not Enforced)';
+    color = 'grey';
+    icon = <LayerGroupIcon />;
+  } else if (acceptedCondition) {
+    labelText = 'Accepted (Not Enforced)';
+    color = 'purple';
+    icon = <UploadIcon />;
+  } else if (conflictedCondition) {
+    labelText = 'Conflicted (Not Accepted)';
+    color = 'red';
+    icon = <ExclamationTriangleIcon />;
+  } else if (targetNotFoundCondition) {
+    labelText = 'TargetNotFound (Not Accepted)';
+    color = 'red';
+    icon = <ExclamationTriangleIcon />;
+  } else if (unknownCondition) {
+    labelText = 'Unknown (Not Enforced)';
+    color = 'orange';
+    icon = <ExclamationTriangleIcon />;
```

Comment:
This shows for below status condition:
```yaml
    - lastTransitionTime: '2024-09-11T11:01:04Z'
      message: 'DNSPolicy has encountered some issues: invalid targetRef.Namespace ingress-gateway. Currently only supporting references to the same namespace'
      reason: Unknown
      status: 'False'
      type: Accepted
```
There's no Enforced status condition, so probably doesn't make sense to say `Not Enforced`.
`Unknown (Not Accepted)` maybe?